src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
 public class Application {
+    private static int countBall = 3;
```

Comment:
it should be constant

Reply:
i can`t make this field a constant because i can`t change the value in the loop